core/controllers/story_editor.py

```diff
@@ -59,6 +59,51 @@ class EditableStoryDataHandler(base.BaseHandler):
     """A data handler for stories which support writing."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'story_id': {
+            'schema': {
+                'type': 'basestring'
+            },
+            'validators': [{
+                'id': 'has_length',
+                'value': constants.STORY_ID_LENGTH
+            }]
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int',
+                    'validators': [{
+                        'id': 'has_length_at_least',
+                        'min_value': 1
+                    }]
```

Comment:
> Don't add default value add a check that it needs to be at least 1.

@vojtechjelinek Please take a look at this.

Reply:
LGTM!